hazelcast-client-legacy/src/test/java/com/hazelcast/client/cluster/ClientClusterServiceMemberListTest.java

```diff
@@ -87,6 +89,13 @@ public void run()
         });
     }
 
+    @Test
+    public void testMemberListOrderConsistentWithServer() {
+        Set<Member> membersFromClient = client.getCluster().getMembers();
+        Set<Member> membersFromServer = dataInstance.getCluster().getMembers();
```

Comment:
I did not call it dataInstance. It was there before I wrote the test. I just used it.